client/src/elm/Utils/NominalDate.elm

```diff
@@ -94,6 +94,47 @@ renderAgeMonthsDaysParts language birthDate now =
     [ monthPart, dayPart ]
 
 
+renderAgeYearsMonths : Language -> NominalDate -> NominalDate -> String
+renderAgeYearsMonths language birthDate now =
+    let
+        diff =
+            diffCalendarYearsAndMonths birthDate now
+
+        months =
+            diff.months
+
+        years =
+            diff.years
+    in
+    case years of
+        0 ->
+            case months of
+                1 ->
+                    translate language <| Translate.AgeSingleMonthWithoutDay months
+
+                _ ->
+                    translate language <| Translate.AgeMonthsWithoutDay months
+
+        1 ->
```

Comment:
@Diama1 

Still wrong. It should be saying one of:

- One year old
- One year and one month old
- One year and X months old